src/Draco.ProjectFile/DracoBuildTask.cs

```diff
@@ -0,0 +1,47 @@
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using Microsoft.Build.Utilities;
+
+namespace Draco.ProjectFile;
+
+public class DracoBuildTask : ToolTask
+{
+    public override bool Execute()
+    {
+        var files = Directory.EnumerateFiles(this.ProjectDirectory, "*.draco", SearchOption.TopDirectoryOnly).Where(x => x == Path.Combine(this.ProjectDirectory, "main.draco"));
+        if (files.Count() == 0)
+        {
+            this.Log.LogError("File main.draco was not found");
+            return false;
+        }
+        var mainFile = files.First();
+        this.ExecuteTool(this.GenerateFullPathToTool(), "", $"compile {mainFile} --output {this.OutputFile} --msbuild-diags");
+        if (this.HasLoggedErrors) return false;
+        return true;
+    }
+
+    protected override string GenerateFullPathToTool() => Path.GetFullPath(Path.Combine(Assembly.GetExecutingAssembly().Location, @"..\..\..\..\..\Draco.Compiler.Cli\bin\Debug\net7.0\Draco.Compiler.Cli.exe"));
```

Comment:
We should try to figure out how we want to deploy the compiler CLI in the SDK, this is very rigid